.github/workflows/release-on-main-pr.yml

```
@@ -0,0 +1,73 @@
+            echo "exists=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Get latest merged PR info
+        id: pr
+        if: steps.tag-check.outputs.exists == 'false'
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          PR_INFO=$(gh pr list --state merged --base main --limit 10 --json number,mergedAt | jq 'sort_by(.mergedAt) | reverse | .[0]')
+          PR_NUMBER=$(echo "$PR_INFO" | jq -r '.number')
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)